service/history/api/signalwithstartworkflow/api.go

```diff
@@ -50,8 +50,8 @@ func Invoke(
 	}
 	namespaceID := namespaceEntry.ID()
 
-	var currentWorkflowContext api.WorkflowContext
-	currentWorkflowContext, err = workflowConsistencyChecker.GetWorkflowContext(
+	var currentWorkflowLease api.WorkflowLease
+	currentWorkflowLease, err = workflowConsistencyChecker.GetWorkflowContext(
```

Comment:
do you mind renaming the `GetWorkflowContext` function?

Reply:
oh, right; great point!